<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTab</class>
 <widget class="QWidget" name="SearchTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui.qrc">
    <normaloff>:/filefind.png</normaloff>:/filefind.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="searchesTab">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Attributes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="attrsTree"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Search options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="nodeBrowserLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QCheckBox" name="selectAll">
             <property name="toolTip">
              <string>Check all results</string>
             </property>
             <property name="text">
              <string>Select all</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="path"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Name contain</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameContain"/>
             </item>
             <item>
              <widget class="QCheckBox" name="caseSensitiveName">
               <property name="text">
                <string>Case sensitive</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="typeName"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="addOption">
               <property name="toolTip">
                <string>Add search clause</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="gui.qrc">
                 <normaloff>:/add.png</normaloff>:/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="advOptBox">
             <property name="title">
              <string>More options</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="margin">
               <number>3</number>
              </property>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents_2">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>584</width>
                   <height>317</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                  <item>
                   <layout class="QGridLayout" name="advancedOptions"/>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="completeClause"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>3</number>
             </property>
             <item>
              <widget class="QProgressBar" name="searchBar">
               <property name="value">
                <number>0</number>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
               <property name="format">
                <string>%p%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalHits">
               <property name="text">
                <string>match(s)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="exportButton">
               <property name="text">
                <string>Export</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopSearchButton">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="launchSearchButton">
               <property name="text">
                <string>Find</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
